src/blocks/scratch3_control.js

```diff
@@ -182,6 +183,10 @@ class Scratch3ControlBlocks {
     incrCounter () {
         this._counter++;
     }
+
+    allAtOnce (args, util) {
```

Comment:
@towerofnix, thanks for this PR! Do you mind adding the comment from scratch-blocks (or the part of it talking about 2.0 functionality) here as well?

Reply:
Will do!